Makefile

```diff
@@ -191,6 +195,10 @@ generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 		output:rbac:dir=$(RBAC_ROOT) \
 		rbac:roleName=manager-role
 
+.PHONY: generate-flavors ## Generate template flavors
+generate-flavors: $(KUSTOMIZE)
+	./hack/gen-flavors.sh
```

Comment:
should we also add flavor generation to the release-manifests target?

Reply:
Should we? If the build is valid, has passed the validate-gen goal, then the flavors have been generated and have not changed.

Reply:
That's true. Should we at least copy them to `$(RELEASE_DIR)` then? Just trying to think of a way to make it easier to automate release artifacts as part of the release process since we'll have to upload those files to the github release... Otherwise we have to go manually fetch them from the templates dir (arguably not that much work but still)...

Reply:
I like cp'ing them into the release dir. I'll add a goal for that.

Reply:
Should be handled with `make release-templates`, which is added to `make release`.